src/Application.vala

```diff
@@ -52,6 +52,7 @@ namespace Maya {
 
             application_id = Build.EXEC_NAME;
 
+            Environment.set_variable ("GTK_USE_PORTAL", "1", true);
```

Comment:
I think the same comment that mail has about "removing this when Gtk4/Gtk.FileDialog is available" should be here too.

Reply:
```suggestion
            // FIXME: Remove once ported to Gtk.FileDialog
            Environment.set_variable ("GTK_USE_PORTAL", "1", true);
```